frontend/src/components/Calendar/index.scss

```
@@ -0,0 +1,40 @@
+.calendar {
+  box-sizing: border-box;
+  margin-top: 30px;
+  margin-bottom: 30px;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  padding: 0px;
+  width: 844px;
+  height: auto;
+
+  background: #fcfcfc;
+  border: 1px solid #ccd3d3;
```

```suggestion
  border: 1px solid $line;
```

제가 _colors.scss 에 명시해 놓은 색이 있는 데 그거 가져다가 쓰면 좋을 거 같아요

---

아! 요건 제가 어제 밤에 정신없이 하다보니 일관성있게 못 둔 것 같습니다. _color.scss에 존재하는 color들은 어제 작업한 영역에 다 들어갈 수 있도록 수정 해놓겠습니다!